website/docs/introduction/git-cheat-sheet.md

```diff
@@ -94,7 +94,7 @@ See `sl help revset` for more ways of referencing commits.
 |Undo commit |git reset --soft HEAD^ |sl uncommit |
 |Undo partial commit |git reset --soft HEAD^ FILE |sl uncommit FILE |
 |Undo amend |git reset HEAD@{1} |sl unamend |
-|Undo rebase/etc |git reset —hard HEAD@{1} |sl undo |
+|Undo rebase/etc |git reset -—hard HEAD@{1} |sl undo |
```

Comment:
I'm looking at this on ReviewStack (https://reviewstack.dev/facebook/sapling/pull/152) and it appears to be non-ASCII?

Reply:
@bolinfest I see, there were previously em-dashes in the file, and I just added a single hyphen, thinking that they were regular dashes. I've updated the commit to instead replace all em-dashes.

Reply:
I checked, and there are only a handful of other en- and em-dashes in the repository, but only in places where it doesn't matter for copy/paste or execution reasons.